config.go

```diff
@@ -526,6 +526,23 @@ func LoadConfig() (*Config, error) {
 		return nil, err
 	}
 
+	// Make sure everything we just loaded makes sense.
+	cleanCfg, err := SanitizeConfig(cfg, usageMessage)
+	if err != nil {
+		return nil, err
+	}
+
+	// Warn about missing config file only after all other configuration is
+	// done.  This prevents the warning on help messages and invalid
+	// options.  Note this should go directly before the return.
+	if configFileError != nil {
+		ltndLog.Warnf("%v", configFileError)
+	}
+
+	return cleanCfg, nil
+}
+
+func SanitizeConfig(cfg Config, usageMessage string) (*Config, error) {
```

Comment:
Does this need to be exported if `LoadConfig` is? Also an alternative name here should be `ValidateConfig`.

Reply:
Yes, because we don't use `LoadConfig` externally. We need to parse a config file that contains more than just the `lnd`'s struct so we write a custom load function. But we want to be able to use the same validation code. Perhaps we don't need to export `LoadConfig` then? :thinking: 
Perhaps it's still useful for mobile?

Reply:
Yeah think it's still useful in other contexts like mobile.